packages/integration-tests/index.js

```diff
@@ -325,17 +343,25 @@ async function setupSeedData(stackName, bucketName, dataDirectory) {
  * @param {string} postfix - string to append to collection name
  * @returns {Promise.<number>} number of collections added
  */
-async function addCollections(stackName, bucketName, dataDirectory, postfix) {
+//TODO: Change path variable name
+async function addCollections(stackName, bucketName, dataDirectory, postfix,
```

Comment:
I think this should be refactored to use an object instead of arguments list and will add that as a separate minor PR